modules/apps/calendar/calendar-web/src/main/resources/META-INF/resources/js/calendar-a11y.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+AUI.add('liferay-calendar-a11y', (A) => {
+	let html = A.CalendarBase.HEADER_TEMPLATE;
+
+	html = html
+		.replace('aria-role="heading"', '')
+		.replace('<div class="yui3-u', '<h1 class="yui3-u')
+		.replace('</div> +', '</h1> +');
+
+	A.CalendarBase.HEADER_TEMPLATE = html;
```

Comment:
A minor SF: we don't need the `html` variable here. You can simplify to:
```js
A.CalendarBase.HEADER_TEMPLATE = A.CalendarBase.HEADER_TEMPLATE.replace(...
```

Reply:
Since the source formatter had troubles with the chaining method, I put it in a variable with a bit of modification.